corehq/apps/export/templates/export/download_export.html

```diff
@@ -57,7 +57,7 @@
     {% registerurl 'prepare_custom_export' domain %}
     {% registerurl 'prepare_form_multimedia' domain %}
 
-    <div id="download-export-form">
+    <div id="download-export-form" class="ko-template">
```

Comment:
How's that work?  Does it stay hidden till initialized or something?

Reply:
yep, exactly: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/hq.helpers.js#L157